docs(repl):documentation about user snippets (#1379)
telerik/blazor-docs: getting-started/repl/user-snippets.md

---

> title: User Snippets

Reviewer: I agree that the "Predefined" prefix differentiates the snippets, but it doesn't provide much additional explanation of what is the kind of snippets. 
Since we are using the snippets that generate code only in the editor, maybe a better name could be Editor snippets or Code Editor snippets.

> title: User Snippets

Author: Thank you both for the feedback! I initially used the name "User Snippets" as the menu in the REPL tool itself is called so. However, "Code Editor Snippets" name sounds more relevant in this scenario. I renamed the article and we may also consider renaming the menu in REPL.

page_title: User Snippets in Telerik REPL for Blazor
description: Telerik REPL for Blazor allows you to login with your Telerik account and save all snippets you run.
slug: blazor-repl-user-snippets
tags: telerik,blazor,repl,login,user,account,save,snippet
published: True
position: 3
---

# Telerik REPL for Blazor User Snippets

**Telerik REPL for Blazor** allows you to login with your Telerik account or register as a new user.

You can do that by clicking on the user icon in the Toolbar. You will be prompted to the Telerik login page where you can enter your credentials.

Once logged in, the Telerik REPL for Blazor will automatically save every snippet you generate to your account. The snippet generation happens upon clicking of the **Share** button in the Toolbar.

You can manage the saved snippets in the **User Snippets** section of the Sidebar:

![User Snippets menu in Telerik REPL for Blazor](../images/repl-user-snippets.png)

> The **User Snippets** menu allows you to perform the following actions with the saved snippets:

Reviewer 2: ```suggestion
The **User Snippets** menu allows you to review and manage your code snippets:
```

> * **Open** - clicking on a snippet will automatically populate it in the editor. You may directly run it or make any further changes. To save the changed snippet, click on the **Share** button - the updated snippet will be saved as a new one.

Reviewer 2: ```suggestion
* **Open**—Clicking a snippet will automatically populate it in the editor. You can run it or make changes. To save the changed snippet, click the **Share** button—the updated snippet will be saved as a new one.
```

> * **Filter** - you can filter the snippets by their name or date of creation.

Reviewer 2: ```suggestion
* **Filter**—You can filter the snippets by name or creation date.
```

* **Sort** - you can sort the snippets in accending or descending order targeting their name or date of creation.

> * **Rename** - by design, the Telerik REPL for Blazor will automatically generate a name when saving the snippet. You can change that to set a meaningful name considering the content of the snippet.

Reviewer 2: ```suggestion
* **Rename**—Telerik REPL for Blazor automatically generates a name when saving the snippet. You can change that default name to something meaningful.
```

> * **Delete** - while you are logged in, the the Telerik REPL for Blazor will automatically save each snippet you generate. The **User Snippets** menu allows you to delete a snippet if it is no longer needed.

Reviewer 2: ```suggestion
* **Delete**—While you are logged in, Telerik REPL for Blazor automatically saves each snippet you generate. The **User Snippets** menu allows you to delete the snippets you no longer need.
```

## See Also

> * [Telerik REPL for Blazor - Snippets and Scaffolders]({%slug blazor-repl-snippets-scaffolders%})

Reviewer 2: ```suggestion
* [Predefined Snippets and Scaffolders in Telerik REPL for Blazor]({%slug blazor-repl-snippets-scaffolders%})
```
